tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature

```diff
@@ -2,30 +2,18 @@
 @api
 @javascript
 
-Feature: MATCH Engagements submissions page
-  The MATCH Engagement Submissions page has a table listing submissions
-  This page should be viewable by match_pm and match_sc roles TODO: but not authenticated
-  Displaying:
-  - Link to edit
-  - Status
-  - Title with link to view
-  - Team with links to community persona for researcher(s), student(s), mentor(s), and consultant
-  - Launch presentation date
-  - Wrap presentation date
+Feature: testing match engagements submissions for Match SC role
 
```

Comment:
hi @alvaradom0321  -- we don't want to lose all these lines of description of what the test should do.